addrmgr/network.go

```diff
@@ -251,7 +249,7 @@ func isValid(netIP net.IP) bool {
 // IsRoutable returns whether or not the passed address is routable over
 // the public internet.  This is true as long as the address is valid and is not
 // in any reserved ranges.
-func IsRoutable(netIP net.IP) bool {
```

Comment:
I think this should remain exported.  It is generally useful and, iirc, used by at least wallet.

Reply:
Updated to re-expose function.